src/Containers/Microsoft.NET.Build.Containers/FallbackToHttpMessageHandler.cs

```diff
@@ -12,13 +12,16 @@ namespace Microsoft.NET.Build.Containers;
 /// </summary>
 internal sealed partial class FallbackToHttpMessageHandler : DelegatingHandler
 {
+    private readonly string _registeryName;
```

Comment:
Typo: all usages of this field should be `_registryName`.

Reply:
Fixed.